test/unit/http-connection.test.ts

```diff
@@ -43,6 +43,8 @@ const options = {
   context: null
 }
 
+const nodeKeepAliveByDefault = Number(process.versions.node.split('.')[0]) >= 19
```

Comment:
Instead of parsing the Node.js version to detect features, you can just check the global agent, which is slightly less brittle:
```
const nodeKeepAliveByDefault = http.globalAgent.keepAlive
```

Reply:
I was trying that and type checking complains `.keepAlive` isn't on the Agent type:

![Screenshot 2023-03-29 at 2 23 02 PM](https://user-images.githubusercontent.com/46866/228670977-3188c107-de7c-4993-be8a-fa47e16ebe77.png)

It is not a documented field on `Agent` at https://nodejs.org/api/http.html.
I'm happy with either. Which would you prefer?

Reply:
Weird, I thought it was! Let's just leave it. It's only happening in the tests, after all. :slightly_smiling_face: